k8s-genie/scripts/provision.sh

```diff
@@ -13,6 +13,9 @@ sed -i '/ swap / s/^/#/' /etc/fstab
 sed -i 's/quiet"/quiet spectre_v2=off nopti hugepagesz=2M hugepages=64"/' /etc/default/grub
 grub2-mkconfig -o /boot/grub2/grub.cfg
 
+# Allow user namespaces
+systemctl -w user.max_user_namespaces=1024
```

Comment:
I couldn't find the `-w` flag in `systemctl` docs. What does it do?
I know that you usually would use `sysctl` for such configurations...